.github/workflows/run_cloudx.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_CLOUDX }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_CLOUDX }}
       #AWS_BUCKET: ${{ secrets.AWS_BUCKET_CLOUDX }} Cloudx doesn't store objects for now
-      OLD_LINE: 'opt-in-not-required'
-      NEW_LINE: 'not-opted-in'
+      OLD_LINE: '== "opt-in-not-required"'
+      NEW_LINE: '!= "not-opted-in"'
```

Comment:
Can we avoid modifying the sh script on the go? I was on PTO when this was merged but I would not have approved this change.

There's more than one reason:
1. Modifying CI script during runtime is not a good practice, it can bring unexpected results.
2. It is difficult to debug if we forget about this sneaky injection in the future, or for someone who is new and has to troubleshoot CI issues.
3. The aws.sh script should be enough to work the same way in both Image Builder and Cloud Experience environments. If this is not the case, then we should not share the same CI infrastructure for this purpose.

So, what I propose is to apply this change directly in the aws.sh cleanup script, and confirm with the Image Builder folks that this is not causing any issues at all from their side, which I am mostly sure it's 100% the case. Before, the build was broken because of other reasons, and I think they already fixed that.

Reply:
Sure, feel free to take over and propose a long term solution for this. 
This is mainly for reporting the issue at hand. I'll leave the implementation up to you guys.

Reply:
Yes, I know you were not the original owner of this change introduced recently. I just wanted to keep record of the importance and possible impact of changes like this. So we avoid them in the future. Thanks for taking care of opening this PR!